internal/cli/atlas_dbusers_update.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"strings"
+
+	atlas "github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+	"github.com/mongodb/mcli/internal/flags"
+	"github.com/mongodb/mcli/internal/json"
+	"github.com/mongodb/mcli/internal/store"
+	"github.com/mongodb/mcli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type atlasDBUsersUpdateOpts struct {
+	*globalOpts
+	username string
+	password string
+	roles    []string
+	store    store.DatabaseUserStore
+}
+
+func (opts *atlasDBUsersUpdateOpts) init() error {
+	if opts.ProjectID() == "" {
+		return errMissingProjectID
+	}
+	var err error
+	opts.store, err = store.New()
+	return err
+}
+
+func (opts *atlasDBUsersUpdateOpts) Run() error {
+	current, err := opts.store.DatabaseUser(opts.orgID, opts.username)
```

Comment:
@andreaangiolillo this is your error, this should be `opts.ProjectID()`

Reply:
Also, de we really need to fetch the record first? is there any value to it?

Reply:
I don’t understand your comment on the pull request. Does the command update an existing record right? so I need to get that record in order to append the new roles to it. what am I missing?

Reply:
How do we know if the username used in the command is associated to a database user if we don't fetch the record first?